ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -222,8 +224,8 @@ public float getFastSyncFullValidationRate() {
     return fastSyncFullValidationRate;
   }
 
-  public int getFastSyncMinimumPeerCount() {
-    return fastSyncMinimumPeerCount;
+  public int getSyncMinimumPeerCount() {
+    return syncMinimumPeerCount;
```

Comment:
this really is a "fast sync" minimum peer count though right?  there is no minimum peer count for full sync AFAIK

Reply:
A number of the `fast` sync options also apply to `snap` and `checkpoint` sync, but because `fast` came long first a number of the variables (and things like the CLI arg `--fast-sync-min-peers`) have `fast` in the name. We've slowly been removing `fast` from them (so for example, the CLI options is now `--sync-min-peers`) but you're right, they're don't apply to `full` sync. I think it's OK that we remove `fast` to avoid confusion with them applying to `snap` and `checkpoint`, but I agree it's not as clear that they don't apply to `full`.

Reply:
IMO unless we can differentiate that the min peer count only applies to these 'fast' sync processes, we are not adding clarity, just changing things. 

 `--state-sync-min-peers` is less painful than `--weak-subjectivity-sync-min-peers`.